documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -60,30 +77,85 @@ spec:
             host: broker-1.myingress.com
           - broker: 2
             host: broker-2.myingress.com
+          class: nginx  
```

Comment:
Should we add here a callout and clarify what it is and that it is optional if the user has set up a default ingress class?